src/Umbraco.Web.UI.Client/src/less/property-editors.less

```diff
@@ -165,6 +165,8 @@
     .sp-replacer {
         display: inline-flex;
         margin-right: 18px;
+        border: solid 1px #d8d7d9;
```

Comment:
Always a difficult one to figure out, but there's a variable for this in variables.less and the color can be replaced with `@gray-8`. Can you please update this one and test out that the correct border color still gets applied?

Reply:
Duh.. i meant to check that and missed out, but I think I have fixed it now :-)